docs/reference/tool-orchestration-auth.ts

```
@@ -0,0 +1,520 @@
+
+      // Publish error event
+      await this.redis.publish(
+        `tenant:${tenant.id}:executions`,
+        JSON.stringify({ executionId, status: 'failed', error: error.message })
+      );
+    }
+  }
+}
+
+// memory-service/src/middleware/AuthMiddleware.ts
+import { Request, Response, NextFunction } from 'express';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Request'. Consider to delete it or rename it.

'Request' is defined here:


(lint/suspicious/noRedeclare)